src/test/unit/cli_unittest.cc

```diff
@@ -116,7 +116,7 @@ TEST(CLIUnittest, TestCliSetArray)
 
 
     //cliGet((char *)"osd_item_vbat");
-    //EXPECT_EQ(false, false);
+    //EXPECT_FALSE(false);
```

Comment:
Maybe is better to remove this if this test is not used anymore.

Reply:
Lol, `EXPECT_EQ(false, false)` wasn't the best test to start with.